game.js

```diff
@@ -36,6 +22,75 @@ const saveStates = (playerState, savedState) => {
   });
 };
 
+const createBoard = (size) => {
+const gameContainer =  document.querySelector('.game');
+gameContainer.classList.remove("hidden");
+if(game.boardSize !== "")
```

Comment:
always use {} inside if statements

Reply:
even if its for just one line?
writing if(...) return .... is better?

Reply:
always use curly braces for consistency

Reply:
ok!